jupyter-extension/jupyterlab_pachyderm/mount_server_client.py

```diff
@@ -88,7 +88,12 @@ async def _ensure_mount_server(self):
                     # Cannot mount in non-privileged container, so use unshare for a private mount
                     get_logger().info("Non-privileged container...")
                     subprocess.run(['mkdir','-p', f'/mnt{self.mount_dir}'])
-                    mount_server_cmd = f"unshare -Ufirm mount-server --mount-dir /mnt{self.mount_dir} --allow-other=false"
+
+                    if self.sock_path:
```

Comment:
Just move 85-86 until after the `if self.nopriv:` conditional.

You are appending to the mount_server_cmd, so no need to do it twice.
The self.nopriv path has to replace the --mount-dir so it couldn't append.

Reply:
It is not clear to me, if I remove 85-86 the code logic changes. At line 92 I already have if self.sock_path condition so it is not needed there.

Reply:
You  don't need the conditional at 92 at all, if you move 85-86 after it.

You are causing unnecessary duplication.